pos_barcode_tare/models/pos_config.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+    tare_label_button = fields.Boolean(
```

Comment:
Generally, all the PoS options names are prefixed by iface.
Think that it should be applied here. (not a blocking point, just a Odoo convention)

Reply:
I'll change that. I'd like to stick to conventions as much as possible.